presto-main/src/test/java/com/facebook/presto/operator/scalar/FunctionAssertions.java

```diff
@@ -209,6 +218,22 @@ public Object selectSingleValue(String projection, Session session)
 
         List<Object> results = new ArrayList<>();
 
+        //
+        // If the projection does not need bound values, execute query using full engine
```

Comment:
Hey,
I know this review is ancient old, but is it possible for you to let me know if this copy of code is intended and what reasoning is behind that?
What I mean is that same check is done later in this function (lines: 257:269), so maybe you just wanted to move this code? It seems that possibly we could reduce testing time by avoiding this copy.
@dain